sourcecode/driver.py

```diff
@@ -0,0 +1,36 @@
+from sourcecode.weatherByYearMonthBar.weatherman import WeatherManBar
+from sourcecode.weatherByYearMonthSingleBar.weatherman import WeatherManSingleBar 
+from sourcecode.weatherByYear.weatherman import WeatherManYear
+from sourcecode.weatherByYearMonth.weatherman import WeatherManYearMonth
+
+class Driver:
```

Comment:
The class which is being called `main` should be called `Driver` and this one needs some proper naming.